src/app/pages/instances/components/all-instances/instance-details/instance-disks/instance-disks.component.html

```diff
@@ -1,3 +1,6 @@
+@let isVmRunning = instance().status === instanceStatus.Running && instance().type === instanceType.Vm;
```

Comment:
Better not to overload template with business logic and have these as computed properties in the component.